website/docs/docs/build/incremental-microbatch.md

```diff
@@ -0,0 +1,147 @@
+---
+title: "About microbatch incremental models"
+description: "Learn about the 'microbatch' strategy for incremental models."
+id: "incremental-microbatch"
+---
+
+# About microbatch incremental models <Lifecycle status="beta" />
+
+:::info Microbatch
+
+The `microbatch` strategy is available in [dbt Cloud Versionless](/docs/dbt-versions/upgrade-dbt-version-in-cloud#versionless) and dbt Core v1.9.
+
+Read and participate in the discussion: [dbt-core#10672](https://github.com/dbt-labs/dbt-core/discussions/10672)
+
+:::
+
+## What is microbatch?
+
+Incremental models in dbt are a [materialization](https://docs.getdbt.com/docs/build/materializations) designed to efficiently update your data warehouse tables by only transforming and loading _new or changed data_ since the last run. Instead of reprocessing an entire dataset every time, incremental models append, update, or replace rows in the existing table with the new data just processed. This can significantly reduce the time and resources required for your data transformations.
```

Comment:
```suggestion
Incremental models in dbt are a [materialization](/docs/build/materializations) designed to efficiently update your data warehouse tables by only transforming and loading _new or changed data_ since the last run. Instead of reprocessing an entire dataset every time, incremental models append, update, or replace rows in the existing table with the new data just processed. This can significantly reduce the time and resources required for your data transformations.
```